api/profile/profileRouter.js

```diff
@@ -383,8 +382,25 @@ router.get('/match/:id', authRequired, (req, res, next) => {
 });
 
 //This route runs to the new DS API endpoint through the checkAvailability middleware that is found in the middleware folders.
-router.post('/availability/:id', checkAvailability, (req, res, next) => {
-  next();
+router.post('/availability/:profile_id', (req, res, next) => {
+  const { profile_id } = req.params;
+  const { accepting_new_mentees } = req.body;
+  console.log({ accepting_new_mentees });
```

Comment:
you'll want to clean up the console.logs--or leave an explanation of why they are here and should remain.

Reply:
Nice catch! Console.logs are removed.